webpack.config.js

```diff
@@ -0,0 +1,42 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+    entry: [
+            'babel-polyfill',
+            './client/src/app.js'
+        ],
+    output: {
+        filename: 'bundle.js',
+        path: path.resolve(__dirname, 'dist')         
+    },    
+    module: {
+        rules: [
+            {   test: '/\.m?js$/',
+                exclude: /(node_modules)/,
+                use: {
+                    loader: 'babel-loader',
+                    options: {
+                        presets: ['@babel/preset-env']
```

Comment:
Lets specify targets for that preset
https://babeljs.io/docs/en/babel-preset-env

Reply:
Either Browser perspective or es6 perspective ?

"targets": {
          "esmodules": true
        }